api/api-template.md

```diff
@@ -0,0 +1,57 @@
+<!-- REMOVE ALL COMMENTS AFTER FILLING OUT TEMPLATE-->
+<!-- Add your function name in the headline below with an accompanying anchor name.  The anchor should be the same as the function name-->
+## function_name() [](function_name) 
```

Comment:
What about labels/badges like :community: or :multinode: anywhere?

Reply:
We don't have those badges yet, but we can easily add them all at once when we decide on which ones we will have.